lib/model/announcement/announcement.dart

```diff
@@ -18,9 +20,18 @@ class Announcement {
         content: json['content'],
         additionalData: (json['additionalData'] ?? <String, dynamic>{})
             as Map<String, dynamic>,
-        startedAt: DateTime.parse(json['startedAt']).millisecondsSinceEpoch,
-        endedAt: DateTime.parse(json['startedAt']).millisecondsSinceEpoch,
+        startedAt: DateTime.parse(json['startedAt']),
+        endedAt: DateTime.parse(json['endedAt']),
       );
 
-  bool get isExpired => DateTime.now().millisecondsSinceEpoch > endedAt;
+  bool get isExpired => DateTime.now().isAfter(endedAt);
+
+  @override
+  String getListTitle() => 'Announcement';
+
+  @override
+  bool isUnread() => false;
```

Comment:
it's better if use isRead instead

Reply:
the UI is focus on unread announce/issue. So i think we should use `isUnread`.

Reply:
I understand, but I think it doesn't related to UI, just a name convention, using `!isRead` is better than `isUnread`

Reply:
But I want to check if announcement is unread. The `isRead` it might be null (in different implementation of ChatThread). So in this case, we have to update all logic where use it.